infrastructure-playbooks/nvme-partition.yml

```diff
@@ -0,0 +1,105 @@
+- name: Create partitions for Bucket Index and FS Journals on a single NVMe device. Prepares for use of osd_scenario=lvm. 
+  hosts:
+  - osds
+
+  tasks:
+
+  - name: Include vars of partition_vars.yaml
+    include_vars:
+      file: partition_vars.yaml
+
+  # need to check if lvm2 is installed
+  - name: install lvm2
+    package:
+      name: lvm2
+      state: present
+
+  # Start Partitioning
+  - name: make Bucket Index partition
```

Comment:
I agree here, partitions don't seem necessary and add a lot to the playbook. It seems like we could take a raw device, make a vg out of it and then make lvs for the bucket index and journal usage. To further simply we could take an input from the user on how many lvs they expect and just split them evenly over the raw device to avoid calculating sizes.